scripts/SANS/sans/command_interface/ISISCommandInterface.py

```diff
@@ -1008,6 +1008,10 @@ def PhiRanges(phis, plot=True):
 def FindBeamCentre(rlow, rupp, MaxIter=10, xstart=None, ystart=None, tolerance=1.251e-4,
                    find_direction=FindDirectionEnum.All, reduction_method=True):
     state = director.process_commands()
+
```

Comment:
is this M, V or P?

Reply:
Hmm I'm not sure. This change isn't really related to the GUI. This file is a set of functions you can call from the python command line to run SANS back-end functions without using the GUI at all. The change here is to bring this interface in line with the old ISISCommandInterface so that legacy scripts will work and is in this PR only because I spotted it whilst testing this one against the command interface.